platform/ui/src/components/Thumbnail/Thumbnail.tsx

```diff
@@ -94,6 +95,12 @@ const Thumbnail = ({
             />
             {` ${numInstances}`}
           </div>
+          {loadingProgress && loadingProgress < 1 && (
+            <div className="mr-4">
+              <span className="text-primary-main font-bold">{'P: '}</span>
```

Comment:
Let's just put the percentage, no `P`, since `S:` here means SeriesNumber. I'm afraid it would confuse people on what `P` is.